core/runtime/src/main/java/io/quarkiverse/langchain4j/runtime/aiservice/AiServiceMethodCreateInfo.java

```diff
@@ -1,7 +1,5 @@
 package io.quarkiverse.langchain4j.runtime.aiservice;
 
-import static org.apache.commons.lang3.StringUtils.EMPTY;
```

Comment:
This dependency has somehow disappeared, but I don't see why we should re-add it just for a constant of an empty string